pipeline-templates/job/deploy.yml

```diff
@@ -1,7 +1,7 @@
 parameters:  
   Environment:
   ServiceConnection:
-  LegacyApimServiceConnection: NAS Pre-Prod (ARM)
+  LegacyApimServiceConnection: NAS-PreProd-ARM (FCS)
```

Comment:
@ChrisJWoodcock why does this service connection needs to be updated?

Reply:
It's been done on master I'll sync the branches